layouts/partials/webpack_url.html

```diff
@@ -1,2 +1,2 @@
-{{ $baseUrl := urls.Parse site.BaseURL }}
-{{- printf "//%s/%s" $baseUrl.Host . -}}
\ No newline at end of file
+{{- $baseUrl := urls.Parse site.BaseURL -}}
+{{- printf "%s://%s" $baseUrl.Scheme (path.Join $baseUrl.Host .) -}}
```

Comment:
This is producing a `"#ZgotmplZ"` value for me indicating that unsafe input is making it to the HTML